0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,82 +1,91 @@
 ---
 title: Interledger Architecture
-draft: 1
+draft: 2
 ---
 # Interledger Architecture
 
 Interledger provides for secure payments across multiple assets on different ledgers. The architecture consists of a conceptual model for interledger payments, a mechanism for securing payments, and a suite of protocols that implement this design.
```

Comment:
Maybe find a different phrasing for 'secure payments' / 'mechanism for securing payments'

Reply:
Hmm, because it's too abstract? What would you say instead?

Reply:
It was because I thought 'secure' referred specifically to 'semi-atomic', but I guess we can still call it secure in a more abstract way, so let's leave it as-is.